src/main/java/com/github/tomakehurst/wiremock/extension/responsetemplating/helpers/JsonData.java

```diff
@@ -158,7 +158,7 @@ public boolean remove(Object o) {
     }
 
     public boolean containsAll(Collection<?> c) {
-      return data.containsAll(c);
+      return new HashSet<>(data).containsAll(c);
```

Comment:
Why?

Reply:
HashSet.containsAll tends to be faster.

Reply:
Of course, but you would need to iterate through all the elements at least once to form the new collection. It defeats the purpose of this change for most of the cases